### PR TITLE
Affpackage doc update

### PR DESCRIPTION
**Pull request by SaraOgaz**
doc update reflecting changes made in astropy/package-template#116

### CODE REVIEW

File: docs/development/affiliated-packages.rst [hunk 1]
```diff
@@ -282,17 +282,15 @@ files manually`_ section since this explains what many of the files do.
    license.
 
 #. Take a moment to look over the ``<packagename>/example_mod.py``,
-   ``<packagename>/tests/test_example.py``, ``scripts/script_example``,
-   and ``<packagename>/example_c.pyx`` files, as well as the
-   ``<packagename>/example_subpkg`` directory. These are examples of a
-   pure-python module, a test script, an example command-line script, a
+   ``<packagename>/tests/test_example.py``, and ``<packagename>/example_c.pyx``
+   files, as well as the ``<packagename>/example_subpkg`` directory.
+   These are examples of a pure-python module, a test script, a
```
Comment (pllim) [inline docs/development/affiliated-packages.rst]: Remove mention of "test script".

File: docs/development/affiliated-packages.rst [hunk 2]
```diff
@@ -269,7 +269,7 @@ files manually`_ section since this explains what many of the files do.
       or just delete it.
    #. Exit out of your text editor.
 
-#. Update the main package docstring in ``<packagename>/__init__.py``.
+#. Update the main package docstring in ``<packagename>/packagename/__init__.py``.
```
Comment (pllim) [inline docs/development/affiliated-packages.rst]: Why the extra layer of `packagename`?

Comment (SaraOgaz) [inline docs/development/affiliated-packages.rst]: __init__.py doesn't live on the upper level anymore it seems...

File: docs/development/affiliated-packages.rst (hunk 1 above)
Comment (SaraOgaz) [inline docs/development/affiliated-packages.rst]: But the test script is still there... that's what the test_example.py is I thought...

Comment (pllim) [inline docs/development/affiliated-packages.rst]: Here, the test script specifically means the one you deleted. The `.py` is "pure-python module". Technically, "python" should also be capitalized but that is not what your PR is set out to do, so you can ignore that part.

File: docs/development/affiliated-packages.rst (hunk 2 above)
Comment (pllim) [inline docs/development/affiliated-packages.rst]: I still see it at https://github.com/astropy/package-template/tree/master/packagename

Comment (SaraOgaz) [inline docs/development/affiliated-packages.rst]: @pllim Yeah, but when you follow the instructions, you download the whole repo as <packagename>, i.e. <packagename> on this doc page corresponds to package-template. And then you go one more folder down into packagename.  I can show you on my local copy if you want...

File: docs/development/affiliated-packages.rst (hunk 1 above)
Comment (SaraOgaz) [inline docs/development/affiliated-packages.rst]: I don't think that's true, why would they refer to the command line scripts twice, per the prior text: ...a test script, an example command-line script... 
Also, it seems like the order of reference to the example matches the order of the file listing
 ``<packagename>/example_mod.py``  = pure-python module
-   ``<packagename>/tests/test_example.py``  =  a test script
``scripts/script_example`` = an example command-line script
-   and ``<packagename>/example_c.pyx`` =  `Cython`_ module
-   ``<packagename>/example_subpkg`` directory = a sub-package

Hence the use of "respectively" at the end of the sentence.

Comment (pllim) [inline docs/development/affiliated-packages.rst]: Apparently, I didn't read it carefully enough. I think you're right. Please ignore the noise!

File: docs/development/affiliated-packages.rst (hunk 2 above)
Comment (pllim) [inline docs/development/affiliated-packages.rst]: Sure. We can talk about meeting offline.

**Comment by pllim** [comment]
I think this is okay to merge. Anyone wants to have a second look?

**Comment by pllim** [comment]
Not sure if this is considered a "bug fix". Tagging 1.2.2 for now. Feel free to change.

File: docs/development/affiliated-packages.rst [hunk 3]
```diff
@@ -269,6 +269,12 @@ files manually`_ section since this explains what many of the files do.
       or just delete it.
    #. Exit out of your text editor.
 
+#. Move the main source directory to reflect the name of your package.
+   To tell your DVCS about this move, you should use it, and not ``mv``
```
Comment (saimn) [inline docs/development/affiliated-packages.rst]: *you should use* **git** I think ?

Comment (pllim) [inline docs/development/affiliated-packages.rst]: @saimn , the actual command does use `git mv`, so not sure what you mean here.

Comment (saimn) [inline docs/development/affiliated-packages.rst]: @pllim - I thought "you should use it" has a typo "it" -> "git", but the "it" can also refer to "your DVCS", so forget my comment, sorry.

Comment (pllim) [inline docs/development/affiliated-packages.rst]: Ah, okay. I think "it" is correct here. Because the next sentence says, "For example, with git..."

**Comment by mhvk** [comment]
I had a look and this all looks good, and clearer than before, so thanks @SaraOgaz!  The milestone seems OK, too, so I'll just merge.

**Pull request merged**